Dockerfile

```diff
@@ -43,94 +46,92 @@ RUN wget http://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz \
     && rm install-tl-unx.tar.gz
 
 # Install SimbaAthena & configure ODBC drivers
-RUN wget -q https://s3.amazonaws.com/athena-downloads/drivers/ODBC/SimbaAthenaODBC_1.0.3/Linux/simbaathena-1.0.3.1004-1.x86_64.rpm -P /tmp \
-    && alien -i /tmp/simbaathena-1.0.3.1004-1.x86_64.rpm \
-    && rm -f /tmp/simbaathena-1.0.3.1004-1.x86_64.rpm
+RUN curl -s "https://s3.amazonaws.com/athena-downloads/drivers/ODBC/SimbaAthenaODBC_${SIMBAATHENA_MAJOR_VERSION}/Linux/simbaathena-${SIMBAATHENA_MAJOR_VERSION}.${SIMBAATHENA_PATCH_VERSION}.x86_64.rpm" > "/tmp/simbaathena.rpm" \
+    && alien -i "/tmp/simbaathena.rpm" \
+    && rm -f "/tmp/simbaathena.rpm"
 COPY files/odbc* /etc/
 
-# Install Miniconda
-RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh \
-    && /bin/bash ~/miniconda.sh -b -p /opt/conda \
-    && rm ~/miniconda.sh \
-    && /opt/conda/bin/conda clean -tipsy \
+# Install Anaconda
+COPY --from=miniconda3 /opt/conda /opt/conda
+RUN /opt/conda/bin/conda clean -tipsy \
     && ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh \
     && echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc
 
-# Configure R
-COPY files/.condarc /opt/conda/
-RUN conda install python=$PY_VERSION r-base=$R_VERSION
-RUN echo '.libPaths(c("~/R/library",paste0(R.home(), "/library"), .libPaths()))' >> /usr/local/lib/R/etc/Rprofile.site \
-    && echo "PATH=\"${PATH}\"" >> /usr/local/lib/R/etc/Renviron \
-    && echo "AWS_DEFAULT_REGION=eu-west-1" >> /usr/local/lib/R/etc/Renviron \
-    && echo "r-libs-user=~/R/library" >> /etc/rstudio/rsession.conf
+# # Configure R
+# COPY files/.condarc /opt/conda/
+# RUN conda install python=$PY_VERSION r-base=$R_VERSION
+# RUN echo '.libPaths(c("~/R/library",paste0(R.home(), "/library"), .libPaths()))' >> /usr/local/lib/R/etc/Rprofile.site \
+#     && echo "PATH=\"${PATH}\"" >> /usr/local/lib/R/etc/Renviron \
+#     && echo "AWS_DEFAULT_REGION=eu-west-1" >> /usr/local/lib/R/etc/Renviron \
+#     && echo "r-libs-user=~/R/library" >> /etc/rstudio/rsession.conf
 
-# Install Conda Packages
-RUN conda config --system --set pip_interop_enabled True \
-    && conda install \
-    boto3 \
-    r-aws.s3 \
-    r-aws.ec2metadata \
-    r-Rcpp \
-    r-base64enc \
-    r-bitops \
-    r-caTools \
-    r-codetools \
-    r-curl \
-    r-devtools \
-    r-digest \
-    r-evaluate \
-    r-formatR \
-    r-highr \
-    r-htmltools \
-    r-httr \
-    r-jsonlite \
-    r-knitr \
-    r-markdown \
-    r-packrat \
-    r-readr \
-    r-reticulate \
-    r-rmarkdown \
-    r-rprojroot \
-    r-shiny \
-    r-stringr \
-    r-s3tools \
-    r-tidyverse \
-    r-webshot \
-    r-xml2 \
-    r-yaml \
-    giflib \
-    cairo \
-    pango \
-    libjpeg-turbo \
-    nodejs \
-    altair \
-    libiconv \
-    nbstripout
+# # Install Conda Packages
+# RUN conda config --system --set pip_interop_enabled True \
```

Comment:
Why all the commented out lines?

Reply:
Ah, just testing why I ended up with Conda 4.8.3